deployment/docker-compose.yml

```diff
@@ -35,6 +35,16 @@ db:
 #  ports:
 #    - "25432:5432"
 
+elasticsearch:
+  # Note you cannot scale if you use conteiner_name
+  container_name: bims-elasticsearch
+  restart: on-failure:5
+  #  read_only: true
+  image: elasticsearch:2.4.2
```

Comment:
I see at the bottom you used `elasticsearch-dsl>=5.0,<6.0`, so use `image: elasticsearch:5.6` for example (relevant images). If you actually want to use image elasticsearch:2.4.2, then perhaps your elasticsearch-dsl needs to be changed for version 2.x too.